app/scripts/views/mixins/flow-events-mixin.js

```diff
@@ -23,6 +25,38 @@ define(function (require, exports, module) {
         flowBeginTime: flowBegin,
         flowId: flowId
       });
+    },
+
+    events: {
+      'click a': '_clickFlowEventsLink',
+      'click input': '_engageFlowEventsForm',
+      'input input': '_engageFlowEventsForm',
+      'keyup input': '_keyupFlowEventsInput',
+      'submit': '_submitFlowEventsForm'
+    },
+
+    _clickFlowEventsLink (event) {
+      if (event && event.target && event.target.id) {
```

Comment:
The KeyUp event handler scares me less than relying on the element `id` attribute. I fear we are going to forget to add an ID to an element if this mixin is ever added to another module. Logging an error seems appropriate if no id can be found. Would using a data attribute instead of the `id` be more "pure" to the purpose of each?